### DISCUSSION TITLE
[Bug]: After marking field as non-editable or invisible in class, it is not reflecting on object form.

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#10030] [Q&A / Support] [answered]
### Expected behavior

After marking field as non-editable or invisible in class, it should be behave as non-editable field or invisible field.

### Actual behavior

After marking field as non-editable or invisible in class, it is not reflecting on object form.

I am able to reproduce this issue on Pimcore demo site : https://demo.pimcore.fun/admin

### Steps to reproduce

1. Create a class.
2. Add some fields of any type.
3. mark some field as non-editable (in my case, I marked input field).
4. mark some field as invisible  (in my case, I marked input field).
5. save the class.
6. Create the object of that particular class.
7. check expected behavior of fields

**Comment by dvesh3** [comment, marked as answer]
@pimcore-developer sorry, works for me. please check if you are not using "Admin Mode" layout which does not consider these properties for fields on object
 
![image](https://user-images.githubusercontent.com/5137917/129327914-baa27229-904e-4cb6-bd6e-1ad1aa76a972.png)

**Comment by ghost** [comment]
@dvesh3 Thanks Devesh for the quick response. Yes, It is working after mode change.
